local_planner/include/local_planner/avoidance_output.h

```diff
@@ -10,7 +10,7 @@ namespace avoidance {
 enum waypoint_choice { hover, tryPath, direct, reachHeight };
 
 struct avoidanceOutput {
-  waypoint_choice waypoint_type;
+  waypoint_choice waypoint_type = hover;
```

Comment:
For those wondering, this fixes an uninitialized memory thing we found running with `valgrind`

Reply:
Could this fix be moved to a separate PR?

Reply:
hmm I guess that would be more correct, it is just tiny and now that I had to squash everything because of the format change I would have to do a revert here and open another PR. So I would probably say it is easier to leave it in here. But as you wish...